.travis.yml

```diff
@@ -1,20 +0,0 @@
-dist: bionic
-language: java
-jdk:
-  - openjdk11
-before_cache:
-  - rm -f $HOME/.gradle/caches/modules-2/modules-2.lock
-  - rm -f $HOME/.gradle/caches/*/plugin-resolution/
-cache:
```

Comment:
Denne tror jeg ikke er i bruk noen steder. Vi benytter github actions for å testbygge og buildspec.yml for bygging på ekstern masking